ui/js/filters.js

```diff
@@ -33,6 +33,45 @@ treeherder.filter('stripHtml', function() {
     };
 });
 
+treeherder.filter('linkifyURLs', function() {
+    return function(input) {
+        var str = input || '';
+        var urlpattern = /(\b(https?):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim;
```

Comment:
I did a little testing with http://www.regexr.com/ and your regex seemed to work well.  only if the protocol was specified, but that's fine.  I think we always do...